backend/client.go

```diff
@@ -46,7 +46,11 @@ func (c *backendClient) ScheduleNewOrchestration(ctx context.Context, orchestrat
 		}
 	}
 	if req.InstanceId == "" {
-		req.InstanceId = uuid.NewString()
+		u, err := uuid.NewRandom()
```

Comment:
Not fully related but changed this to the API that supports error handling. `uuid.NewString()` could panic if the kernel doesn't have enough entropy.